packages/wonder-blocks-dropdown/src/index.js

```diff
@@ -17,4 +18,4 @@ export {
     MultiSelect,
 };
 
-export type {Labels};
+export type {Labels, SingleSelectLabels};
```

Comment:
This still gets me thinking... should we rename `Labels` to `MultiSelectLabels` to be more explicit and differentiate with the SingleSelect ones?

Reply:
Hmm... we probably should! Might as well wait until the next major release for that, at least.